### DISCUSSION TITLE
APA style refnotes

### DISCUSSION THREAD

**Discussion by OdinJorna** [kookma/Refnotes#10] [Ideas]
In continuation of this [discussion on the google groups](https://groups.google.com/g/tiddlywiki/c/zHxkQyESUSg/m/NfOQcR0mBQAJ). I've read that you're planning on updating the plugin, so I am sharing some information on my use-case in case that is useful for the development.

It is possible to hack the current macro to display journal articles in APA style. But the limitation is that all references are displayed in the same configuration. An addition to the macro that recognises the type of reference and adjusts the presentation of the reference would be really useful. Especially when in practice, references come from different types of sources/types.

When I changed the inline reference to APA style (Author, Year), I ran into the issue with some references that have multiple authors. It is common to write only the first author when there are more than 5 to avoid cluttering. So instead of (author1, author2, author3, author4, author5, 2020), one would just write (author1 et al., 2020).

If you like some help with APA or with testing, I'd love to help out.

**Comment by kookma** [comment]
@OdinJorna - Thank you for your feedback. I add this as an enhancement.
